showcase/app/styles/showcase-pages/side-nav.scss

```diff
@@ -79,4 +86,21 @@ body.components-side-nav {
       transform: none !important;
     }
   }
+
+  .shw-component-dropdown-states-matrix {
```

Comment:
[issue] this is a duplicate of the `Dropdown` but the name should be updated or the two class names will conflict in the generated CSS

Reply:
Oops, I missed that. Will update.

Reply:
updated